apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetManifest.java

```diff
@@ -342,21 +358,29 @@ private DataSetManifestEntry() {
       this.exportType = null;
     }
 
-    /** @return the {@link DataSetManifest} that this {@link DataSetManifestEntry} is a part of */
+    /**
+     * Gets the {@link #parentManifest}.
+     *
+     * @return the {@link DataSetManifest} that this {@link DataSetManifestEntry} is a part of
+     */
     public DataSetManifest getParentManifest() {
       return parentManifest;
     }
 
     /**
+     * Gets the {@link #name}.
+     *
      * @return the name of the S3 object/file that is represented by this {@link
      *     DataSetManifestEntry}, which is effectively a relative S3 key (relative to this <code>
-     *     manifest.xml</code> object's key, that is)
+     *          manifest.xml</code> object's key, that is)
```

Comment:
Educate me, would ya? Is this whitespace intentional?

Reply:
No, probably some mistake.